### ISSUE TITLE
Failed validation: #/definitions/SearchResult

### ISSUE THREAD

**Issue by AntonPaskar** [gadicc/node-yahoo-finance2#669]
# Validation Error

## Symbol(s) that it happened for
- 
- 
- 

## Error Message

```
This may happen intermittently and you should catch errors appropriately.
@verona/backend: However:  1) if this recently started happening on every request for a symbol
@verona/backend: that used to work, Yahoo may have changed their API.  2) If this happens on
@verona/backend: every request for a symbol you've never used before, but not for other
@verona/backend: symbols, you've found an edge-case.  Please see if anyone has reported
@verona/backend: this previously:
@verona/backend:   https://github.com/gadicc/node-yahoo-finance2/issues?q=is%3Aissue+Failed%20validation%3A%20%23%2Fdefinitions%2FSearchResult
@verona/backend: or open a new issue (and mention the symbol):
@verona/backend:   https://github.com/gadicc/node-yahoo-finance2/issues/new?labels=bug%2C+validation&template=validation.md&title=Failed%20validation%3A%20%23%2Fdefinitions%2FSearchResult
@verona/backend: For information on how to turn off the above logging or skip these errors,
@verona/backend: see https://github.com/gadicc/node-yahoo-finance2/tree/devel/docs/validation.md.
@verona/backend: The following result did not validate with schema: #/definitions/SearchResult
@verona/backend: [
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/quoteType",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooEquity/properties/quoteType/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "EQUITY"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "FUTURE"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/typeDisp",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooEquity/properties/typeDisp/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "Equity"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "Futures"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/quoteType",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooOption/properties/quoteType/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "OPTION"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "FUTURE"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/typeDisp",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooOption/properties/typeDisp/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "Option"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "Futures"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/quoteType",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooETF/properties/quoteType/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "ETF"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "FUTURE"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/typeDisp",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooETF/properties/typeDisp/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "ETF"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "Futures"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/quoteType",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooFund/properties/quoteType/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "MUTUALFUND"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "FUTURE"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/typeDisp",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooFund/properties/typeDisp/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "Fund"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "Futures"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/quoteType",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooIndex/properties/quoteType/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "INDEX"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "FUTURE"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/typeDisp",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooIndex/properties/typeDisp/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "Index"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "Futures"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/quoteType",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooCurrency/properties/quoteType/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "CURRENCY"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "FUTURE"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/typeDisp",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooCurrency/properties/typeDisp/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "Currency"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "Futures"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/quoteType",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooCryptocurrency/properties/quoteType/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "CRYPTOCURRENCY"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "FUTURE"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/typeDisp",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooCryptocurrency/properties/typeDisp/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "Cryptocurrency"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "Futures"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteNonYahoo/required",
@verona/backend:     "keyword": "required",
@verona/backend:     "params": {
@verona/backend:       "missingProperty": "name"
@verona/backend:     },
@verona/backend:     "message": "must have required property 'name'",
@verona/backend:     "data": {
@verona/backend:       "exchange": "CMX",
@verona/backend:       "shortname": "Aluminum Futures,Nov-2023",
@verona/backend:       "quoteType": "FUTURE",
@verona/backend:       "symbol": "ALI=F",
@verona/backend:       "index": "quotes",
@verona/backend:       "score": 3199700,
@verona/backend:       "typeDisp": "Futures",
@verona/backend:       "exchDisp": "New York Commodity Exchange",
@verona/backend:       "isYahooFinance": true
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteNonYahoo/required",
@verona/backend:     "keyword": "required",
@verona/backend:     "params": {
@verona/backend:       "missingProperty": "permalink"
@verona/backend:     },
@verona/backend:     "message": "must have required property 'permalink'",
@verona/backend:     "data": {
@verona/backend:       "exchange": "CMX",
@verona/backend:       "shortname": "Aluminum Futures,Nov-2023",
@verona/backend:       "quoteType": "FUTURE",
@verona/backend:       "symbol": "ALI=F",
@verona/backend:       "index": "quotes",
@verona/backend:       "score": 3199700,
@verona/backend:       "typeDisp": "Futures",
@verona/backend:       "exchDisp": "New York Commodity Exchange",
@verona/backend:       "isYahooFinance": true
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteNonYahoo/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "exchange"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "exchange": "CMX",
@verona/backend:       "shortname": "Aluminum Futures,Nov-2023",
@verona/backend:       "quoteType": "FUTURE",
@verona/backend:       "symbol": "ALI=F",
@verona/backend:       "index": "quotes",
@verona/backend:       "score": 3199700,
@verona/backend:       "typeDisp": "Futures",
@verona/backend:       "exchDisp": "New York Commodity Exchange",
@verona/backend:       "isYahooFinance": true
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteNonYahoo/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "shortname"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "exchange": "CMX",
@verona/backend:       "shortname": "Aluminum Futures,Nov-2023",
@verona/backend:       "quoteType": "FUTURE",
@verona/backend:       "symbol": "ALI=F",
@verona/backend:       "index": "quotes",
@verona/backend:       "score": 3199700,
@verona/backend:       "typeDisp": "Futures",
@verona/backend:       "exchDisp": "New York Commodity Exchange",
@verona/backend:       "isYahooFinance": true
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteNonYahoo/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "quoteType"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "exchange": "CMX",
@verona/backend:       "shortname": "Aluminum Futures,Nov-2023",
@verona/backend:       "quoteType": "FUTURE",
@verona/backend:       "symbol": "ALI=F",
@verona/backend:       "index": "quotes",
@verona/backend:       "score": 3199700,
@verona/backend:       "typeDisp": "Futures",
@verona/backend:       "exchDisp": "New York Commodity Exchange",
@verona/backend:       "isYahooFinance": true
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteNonYahoo/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "symbol"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "exchange": "CMX",
@verona/backend:       "shortname": "Aluminum Futures,Nov-2023",
@verona/backend:       "quoteType": "FUTURE",
@verona/backend:       "symbol": "ALI=F",
@verona/backend:       "index": "quotes",
@verona/backend:       "score": 3199700,
@verona/backend:       "typeDisp": "Futures",
@verona/backend:       "exchDisp": "New York Commodity Exchange",
@verona/backend:       "isYahooFinance": true
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteNonYahoo/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "score"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "exchange": "CMX",
@verona/backend:       "shortname": "Aluminum Futures,Nov-2023",
@verona/backend:       "quoteType": "FUTURE",
@verona/backend:       "symbol": "ALI=F",
@verona/backend:       "index": "quotes",
@verona/backend:       "score": 3199700,
@verona/backend:       "typeDisp": "Futures",
@verona/backend:       "exchDisp": "New York Commodity Exchange",
@verona/backend:       "isYahooFinance": true
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteNonYahoo/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "typeDisp"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "exchange": "CMX",
@verona/backend:       "shortname": "Aluminum Futures,Nov-2023",
@verona/backend:       "quoteType": "FUTURE",
@verona/backend:       "symbol": "ALI=F",
@verona/backend:       "index": "quotes",
@verona/backend:       "score": 3199700,
@verona/backend:       "typeDisp": "Futures",
@verona/backend:       "exchDisp": "New York Commodity Exchange",
@verona/backend:       "isYahooFinance": true
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteNonYahoo/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "exchDisp"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "exchange": "CMX",
@verona/backend:       "shortname": "Aluminum Futures,Nov-2023",
@verona/backend:       "quoteType": "FUTURE",
@verona/backend:       "symbol": "ALI=F",
@verona/backend:       "index": "quotes",
@verona/backend:       "score": 3199700,
@verona/backend:       "typeDisp": "Futures",
@verona/backend:       "exchDisp": "New York Commodity Exchange",
@verona/backend:       "isYahooFinance": true
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/isYahooFinance",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteNonYahoo/properties/isYahooFinance/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": false
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": true
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0/typeDisp",
@verona/backend:     "schemaPath": "#/definitions/SearchQuoteYahooFuture/properties/typeDisp/const",
@verona/backend:     "keyword": "const",
@verona/backend:     "params": {
@verona/backend:       "allowedValue": "Future"
@verona/backend:     },
@verona/backend:     "message": "must be equal to constant",
@verona/backend:     "data": "Futures"
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/quotes/0",
@verona/backend:     "schemaPath": "#/properties/quotes/items/anyOf",
@verona/backend:     "keyword": "anyOf",
@verona/backend:     "params": {},
@verona/backend:     "message": "must match a schema in anyOf",
@verona/backend:     "data": {
@verona/backend:       "exchange": "CMX",
@verona/backend:       "shortname": "Aluminum Futures,Nov-2023",
@verona/backend:       "quoteType": "FUTURE",
@verona/backend:       "symbol": "ALI=F",
@verona/backend:       "index": "quotes",
@verona/backend:       "score": 3199700,
@verona/backend:       "typeDisp": "Futures",
@verona/backend:       "exchDisp": "New York Commodity Exchange",
@verona/backend:       "isYahooFinance": true
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/news/0",
@verona/backend:     "schemaPath": "#/definitions/SearchNews/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "thumbnail"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "uuid": "83079b37-974a-3ff9-b0b4-5991bb422ace",
@verona/backend:       "title": "Apple's new iPhone 15 series gets mixed reaction from some Chinese consumers amid hype over Huawei's latest 5G handsets and government use ban",
@verona/backend:       "publisher": "South China Morning Post",
@verona/backend:       "link": "https://finance.yahoo.com/news/apples-iphone-15-series-gets-093000233.html",
@verona/backend:       "providerPublishTime": "2023-09-14T09:30:00.000Z",
@verona/backend:       "type": "STORY",
@verona/backend:       "thumbnail": {
@verona/backend:         "resolutions": [
@verona/backend:           {
@verona/backend:             "url": "https://s.yimg.com/uu/api/res/1.2/S59A_jt66Hi5dt92UqVMFQ--~B/aD0xMzMxO3c9MjAwMDthcHBpZD15dGFjaHlvbg--/https://media.zenfs.com/en/south_china_morning_post_us_228/2886d3bad59bd43cef127f9293f1cd43",
@verona/backend:             "width": 2000,
@verona/backend:             "height": 1331,
@verona/backend:             "tag": "original"
@verona/backend:           },
@verona/backend:           {
@verona/backend:             "url": "https://s.yimg.com/uu/api/res/1.2/OtSIpiXDrm7WRJx3DVG_Lg--~B/Zmk9ZmlsbDtoPTE0MDtweW9mZj0wO3c9MTQwO2FwcGlkPXl0YWNoeW9u/https://media.zenfs.com/en/south_china_morning_post_us_228/2886d3bad59bd43cef127f9293f1cd43",
@verona/backend:             "width": 140,
@verona/backend:             "height": 140,
@verona/backend:             "tag": "140x140"
@verona/backend:           }
@verona/backend:         ]
@verona/backend:       },
@verona/backend:       "relatedTickers": [
@verona/backend:         "AAPL"
@verona/backend:       ]
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/news/0",
@verona/backend:     "schemaPath": "#/definitions/SearchNews/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "relatedTickers"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "uuid": "83079b37-974a-3ff9-b0b4-5991bb422ace",
@verona/backend:       "title": "Apple's new iPhone 15 series gets mixed reaction from some Chinese consumers amid hype over Huawei's latest 5G handsets and government use ban",
@verona/backend:       "publisher": "South China Morning Post",
@verona/backend:       "link": "https://finance.yahoo.com/news/apples-iphone-15-series-gets-093000233.html",
@verona/backend:       "providerPublishTime": "2023-09-14T09:30:00.000Z",
@verona/backend:       "type": "STORY",
@verona/backend:       "thumbnail": {
@verona/backend:         "resolutions": [
@verona/backend:           {
@verona/backend:             "url": "https://s.yimg.com/uu/api/res/1.2/S59A_jt66Hi5dt92UqVMFQ--~B/aD0xMzMxO3c9MjAwMDthcHBpZD15dGFjaHlvbg--/https://media.zenfs.com/en/south_china_morning_post_us_228/2886d3bad59bd43cef127f9293f1cd43",
@verona/backend:             "width": 2000,
@verona/backend:             "height": 1331,
@verona/backend:             "tag": "original"
@verona/backend:           },
@verona/backend:           {
@verona/backend:             "url": "https://s.yimg.com/uu/api/res/1.2/OtSIpiXDrm7WRJx3DVG_Lg--~B/Zmk9ZmlsbDtoPTE0MDtweW9mZj0wO3c9MTQwO2FwcGlkPXl0YWNoeW9u/https://media.zenfs.com/en/south_china_morning_post_us_228/2886d3bad59bd43cef127f9293f1cd43",
@verona/backend:             "width": 140,
@verona/backend:             "height": 140,
@verona/backend:             "tag": "140x140"
@verona/backend:           }
@verona/backend:         ]
@verona/backend:       },
@verona/backend:       "relatedTickers": [
@verona/backend:         "AAPL"
@verona/backend:       ]
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/news/1",
@verona/backend:     "schemaPath": "#/definitions/SearchNews/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "relatedTickers"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "uuid": "45830bcb-9980-3342-bcdc-1c4700427038",
@verona/backend:       "title": "Apple’s China headache and Alibaba’s surprise shake-up",
@verona/backend:       "publisher": "Financial Times",
@verona/backend:       "link": "https://finance.yahoo.com/m/45830bcb-9980-3342-bcdc-1c4700427038/apple%E2%80%99s-china-headache-and.html",
@verona/backend:       "providerPublishTime": "2023-09-14T08:28:13.000Z",
@verona/backend:       "type": "STORY",
@verona/backend:       "relatedTickers": [
@verona/backend:         "AAPL",
@verona/backend:         "BABA"
@verona/backend:       ]
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/news/2",
@verona/backend:     "schemaPath": "#/definitions/SearchNews/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "thumbnail"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "uuid": "82f7f870-12e9-3b7c-a401-025184c0e053",
@verona/backend:       "title": "Germany says France's iPhone 12 process has guiding function for Europe",
@verona/backend:       "publisher": "Reuters",
@verona/backend:       "link": "https://finance.yahoo.com/news/germany-says-frances-iphone-12-081530947.html",
@verona/backend:       "providerPublishTime": "2023-09-14T08:15:30.000Z",
@verona/backend:       "type": "STORY",
@verona/backend:       "thumbnail": {
@verona/backend:         "resolutions": [
@verona/backend:           {
@verona/backend:             "url": "https://s.yimg.com/uu/api/res/1.2/aLv09Nr6xZKUbpIGkYVtQA--~B/aD01Mjg7dz04MDA7YXBwaWQ9eXRhY2h5b24-/https://media.zenfs.com/en/reuters-finance.com/f974ecc8aa2849d3bea3ea696bf3b02c",
@verona/backend:             "width": 800,
@verona/backend:             "height": 528,
@verona/backend:             "tag": "original"
@verona/backend:           },
@verona/backend:           {
@verona/backend:             "url": "https://s.yimg.com/uu/api/res/1.2/yCmZPXndA1exkCVJcu6v1Q--~B/Zmk9ZmlsbDtoPTE0MDtweW9mZj0wO3c9MTQwO2FwcGlkPXl0YWNoeW9u/https://media.zenfs.com/en/reuters-finance.com/f974ecc8aa2849d3bea3ea696bf3b02c",
@verona/backend:             "width": 140,
@verona/backend:             "height": 140,
@verona/backend:             "tag": "140x140"
@verona/backend:           }
@verona/backend:         ]
@verona/backend:       },
@verona/backend:       "relatedTickers": [
@verona/backend:         "AAPL"
@verona/backend:       ]
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/news/2",
@verona/backend:     "schemaPath": "#/definitions/SearchNews/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "relatedTickers"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "uuid": "82f7f870-12e9-3b7c-a401-025184c0e053",
@verona/backend:       "title": "Germany says France's iPhone 12 process has guiding function for Europe",
@verona/backend:       "publisher": "Reuters",
@verona/backend:       "link": "https://finance.yahoo.com/news/germany-says-frances-iphone-12-081530947.html",
@verona/backend:       "providerPublishTime": "2023-09-14T08:15:30.000Z",
@verona/backend:       "type": "STORY",
@verona/backend:       "thumbnail": {
@verona/backend:         "resolutions": [
@verona/backend:           {
@verona/backend:             "url": "https://s.yimg.com/uu/api/res/1.2/aLv09Nr6xZKUbpIGkYVtQA--~B/aD01Mjg7dz04MDA7YXBwaWQ9eXRhY2h5b24-/https://media.zenfs.com/en/reuters-finance.com/f974ecc8aa2849d3bea3ea696bf3b02c",
@verona/backend:             "width": 800,
@verona/backend:             "height": 528,
@verona/backend:             "tag": "original"
@verona/backend:           },
@verona/backend:           {
@verona/backend:             "url": "https://s.yimg.com/uu/api/res/1.2/yCmZPXndA1exkCVJcu6v1Q--~B/Zmk9ZmlsbDtoPTE0MDtweW9mZj0wO3c9MTQwO2FwcGlkPXl0YWNoeW9u/https://media.zenfs.com/en/reuters-finance.com/f974ecc8aa2849d3bea3ea696bf3b02c",
@verona/backend:             "width": 140,
@verona/backend:             "height": 140,
@verona/backend:             "tag": "140x140"
@verona/backend:           }
@verona/backend:         ]
@verona/backend:       },
@verona/backend:       "relatedTickers": [
@verona/backend:         "AAPL"
@verona/backend:       ]
@verona/backend:     }
@verona/backend:   },
@verona/backend:   {
@verona/backend:     "instancePath": "/news/3",
@verona/backend:     "schemaPath": "#/definitions/SearchNews/additionalProperties",
@verona/backend:     "keyword": "additionalProperties",
@verona/backend:     "params": {
@verona/backend:       "additionalProperty": "relatedTickers"
@verona/backend:     },
@verona/backend:     "message": "must NOT have additional properties",
@verona/backend:     "data": {
@verona/backend:       "uuid": "5cf263f8-b41b-3e79-a8cb-07823e2a0e3f",
@verona/backend:       "title": "UPDATE 1-Belgium reviews Apple's iPhone 12 after France halts sales over radiation",
@verona/backend:       "publisher": "Reuters",
@verona/backend:       "link": "https://finance.yahoo.com/news/1-belgium-reviews-apples-iphone-080302109.html",
@verona/backend:       "providerPublishTime": "2023-09-14T08:03:02.000Z",
@verona/backend:       "type": "STORY",
@verona/backend:       "relatedTickers": [
@verona/backend:         "AAPL"
@verona/backend:       ]
@verona/backend:     }
@verona/backend:   }
@verona/backend: ]
```

## Environment
Browser or Node: 
Node version (if applicable): 
Npm version: 
Browser verion (if applicable):
Library version (e.g. `1.10.1`):

**Comment by z3nful** [comment]
What was the search query you used so I can reproduce the response? I can get it sorted out tomorrow with it

**Comment by AntonPaskar** [comment]
It's fixed on version 2.4.6
